backend/src/main/java/ca/bc/gov/app/exception/InvalidRoleException.java

```diff
@@ -6,7 +6,11 @@
 
 @ResponseStatus(HttpStatus.FORBIDDEN)
 public class InvalidRoleException extends ResponseStatusException {
+
+  private static final long serialVersionUID = 6641972355164041184L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data